torch/csrc/profiler/combined_traceback.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <torch/csrc/jit/runtime/interpreter.h>
+#include <torch/csrc/profiler/unwind/unwind.h>
+
+struct PyCodeObject;
+
+namespace torch {
+
+struct TORCH_API SymbolizedTracebacks {
+  std::vector<unwind::Frame> all_frames;
+  // index into all_frames, so that
+  // it is possible to dedupe frame objects in
+  // construction of python objects
+  std::vector<std::vector<uint64_t>> tracebacks;
```

Comment:
what exactly is the nested vector here?

Reply:
oh, it's multiple tracebacks, and then each traceback is indexes into all frames